packages/rrweb/src/replay/index.ts

```diff
@@ -148,8 +146,16 @@ export class Replayer {
 
   private newDocumentQueue: addedNodeMutation[] = [];
 
-  private mousePos: mouseMovePos | null = null;
-  private touchActive: boolean | null = null;
+  // Map of touch ID to the unique vars used to show gestures
+  private multiTouchMap: Record<
```

Comment:
what about something more generic like `pointers` since this handles mouse + touch? (I associate pointers with mouse, but elsewhere we use pointerType, so let's stick with that as mouse + touch).